### ISSUE TITLE
xml response payload parsing fails

### ISSUE THREAD

**Issue by pzieminski** [jarrodek/ChromeRestClient#719]
Your description here
this is the xml

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<feed xmlns="http://www.w3.org/2005/Atom"
      xml:base="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/">
  <id>case-roles</id>
  <title>Case Roles</title>
  <author>
    <name>NGIS</name>
  </author>
  <updated>2016-07-10T18:20:53.020Z</updated>
  <link href="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/" rel="self"/>
  <entry>
    <id>owner</id>
    <title>owner</title>
    <content type="application/vnd.emc.ngis+xml"
             src="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/owner/"/>
    <summary>case-roles</summary>
    <updated>2016-07-10T18:20:53.020Z</updated>
    <link href="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/owner/"
          rel="alternate"/>
    <link href="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/owner/"
          rel="self"/>
  </entry>
  <entry>
    <id>agent</id>
    <title>agent</title>
    <content type="application/vnd.emc.ngis+xml"
             src="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/agent/"/>
    <summary>case-roles</summary>
    <updated>2016-07-10T18:20:53.020Z</updated>
    <link href="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/agent/"
          rel="alternate"/>
    <link href="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/agent/"
          rel="self"/>
  </entry>
  <entry>
    <id>borrower</id>
    <title>borrower</title>
    <content type="application/vnd.emc.ngis+xml"
             src="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/borrower/"/>
    <summary>case-roles</summary>
    <updated>2016-07-10T18:20:53.020Z</updated>
    <link href="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/borrower/"
          rel="alternate"/>
    <link href="/spaces/b79d4fc2-4610-42ba-9830-1e2e6edf9432/case-instance-nodes/924c1568fb8348dd8771753fb1c45c2c/case-roles/borrower/"
          rel="self"/>
  </entry>
</feed>

```
## Expected outcome
What should happen?

xml should be pretty printed in xml preview

## Actual outcome
What happened?

error is displayed in xml preview pane

```
There was an error parsing XML data

NAMESPACE_ERR: The namespace declaration is incorrect
```

## Versions
App: 7.20.37.203-stable
Chrome: 51.0.2704.103
Platform: Windows (7)

## Steps to reproduce
1. 
2. 
3.

**Comment by jarrodek** [comment]
Hi,

Thank you for issue report. It's a duplicate of #691 - the namespace declaration in your XML is invalid. Check the referenced issue for an instructions of how to fix your file.

**Comment by pzieminski** [comment]
This is a well formed xml that used to display correctly in the earlier version of the software and displays fine everywhere else. We can't correct the namespace because we don't own the back end that sends it. Can't you just turn off namespace awareness in the parser? The point is just to pretty print it no?

**Comment by pzieminski** [comment]
This does seem to be a legitimate bug in the software. The xml and xmlns namespaces need not to be explicitly declared as per the xml spec

https://www.w3.org/TR/xml-names11/#ns-decl

**Comment by pzieminski** [comment]
The specific attribute causing the error is define here
https://www.w3.org/TR/xmlbase/

So the xml above is correct

**Comment by jarrodek** [comment]
Hi,

Thanks for clarification. I must have missed this in the spec. 
The issue was caused by the XML parsing library (external library) which seems to be abandoned so reporting an issue there doesn't have sense. 
I've altered local copy of the library and it's now accepting `xml:base` declaration. I've also turned off error checking for the parser so similar situation shouldn't happen again.

Expect the update in coming days.
